src/stim/py/stim.pybind.cc

```diff
@@ -111,7 +111,7 @@ pybind11::object raw_format_data_solo(const FileFormatData &data) {
     result["parse_example"] = data.help_python_parse;
     result["save_example"] = data.help_python_save;
     result["help"] = data.help;
-    return result;
+    return std::move(result);
```

Comment:
Does this move do anything that wouldn't happen automatically?

Reply:
Clang12 brought this to my attention - copy elision won't happen since the return type is of the function is different to the type of `result`.